gw_eccentricity/eccDefinition.py

```diff
@@ -685,16 +682,23 @@ def get_amp_phase_omega_gw(self):
         system is nonprecessing.
         """
         if not self.precessing:
-            amp_gw, phase_gw, omega_gw = (self.dataDict["amplm"][(2, 2)],
-                                          self.dataDict["phaselm"][(2, 2)],
-                                          self.dataDict["omegalm"][(2, 2)])
+            amp_gw, phase_gw, omega_gw = (data_dict["amplm"][(2, 2)],
+                                          data_dict["phaselm"][(2, 2)],
+                                          data_dict["omegalm"][(2, 2)])
```

Comment:
Was there are reason for changing from self.dataDict to data_dict?

Reply:
Yes, we need to call this function inside the process_data_dict before we assign dataDict to self. There we need to provide the dataDict as an argument to this function which gives the amp_gw, phase_gw, omega_gw. Also see related #221 which I will address after this.

Reply:
Can we merge this?